examples/contracts/src/lib.rs

```diff
@@ -120,8 +121,8 @@ mod tests {
             .await?;
         // ANCHOR_END: contract_call_cost_estimation
 
-        let expected_script_gas = 2498;
-        let expected_total_gas = 8750;
+        let expected_script_gas = 2488;
+        let expected_total_gas = 8740;
```

Comment:
It's weird to see this change seeing as the change doesn't impact execution. 

Did you maybe use a different compiler version from the one we have in CI?